Sources/ManageRegistration.php

```diff
@@ -292,20 +322,33 @@ function SetReserved()
 function ModifyRegistrationSettings($return_config = false)
 {
 	global $txt, $context, $scripturl, $modSettings, $sourcedir;
+	global $language, $boarddir;
 
 	// This is really quite wanting.
 	require_once($sourcedir . '/ManageServer.php');
 
+	// Do we have at least default versions of the agreement and privacy policy?
+	$agreement = file_exists($boarddir . '/agreement.' . $language . '.txt') || file_exists($boarddir . '/agreement.txt');
+	$policy = !empty($modSettings['policy_' . $language]);
+
 	$config_vars = array(
 		array('select', 'registration_method', array($txt['setting_registration_standard'], $txt['setting_registration_activate'], $txt['setting_registration_approval'], $txt['setting_registration_disabled'])),
+		array('check', 'enableOpenID'),
+		array('check', 'notify_new_registration'),
 		array('check', 'send_welcomeEmail'),
-		'',
-
-		array('int', 'coppaAge', 'subtext' => $txt['zero_to_disable'], 'onchange' => 'checkCoppa();', 'onkeyup' => 'checkCoppa();'),
+	'',
+		array('check', 'requireAgreement', 'text_label' => $txt['admin_agreement'], 'value' => 1, 'disabled' => 1),
+		array('warning', empty($agreement) ? 'error_no_agreement' : ''),
+		array('check', 'requirePolicyAgreement', 'text_label' => $txt['admin_privacy_policy'], 'value' => 1, 'disabled' => 1),
+		array('warning', empty($policy) ? 'error_no_privacy_policy' : ''),
+	'',
+		array('int', 'coppaAge', 'subtext' => $txt['setting_coppaAge_desc'], 'onchange' => 'checkCoppa();'),
```

Comment:
Instead of creating a new text string, please preserve the previous coppaAge setting definition line that uses `$txt['zero_to_disable']`